test/powershell/Language/Scripting/DeserializedTypeConversion.Tests.ps1

```diff
@@ -0,0 +1,271 @@
+﻿Describe "Tests conversion of deserialized types to original type using object properties." -Tags "CI" {
+
+    BeforeAll {
+    # Test Import/Export Clixml scenario.
+    $tempFile = [System.IO.Path]::GetTempFileName()
```

Comment:
fixed

---
In reply to: [76680334](https://github.com/Powershell/Powershell/pull/2118#discussion_r76680334) [](ancestors = 76680334)